src/components/ProgressBar/ProgressBarStyled.jsx

```diff
@@ -0,0 +1,124 @@
+import styled from "styled-components";
+
+import { MAIN_PAGE_GREY_TEXT } from "../../assets/themes/colors";
+
+export const WeekTasksData = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    gap: 12px;
+
+    @media (max-width: 1000px){
+        align-items: center;
+    }
+
+    @media (max-width: 768px){
+        align-items: center;
+    }
+` 
+
+export const WeekTasksTextElemSet = styled.div`
+    display: flex;
+    gap: 12px;
+
+    @media (max-width: 400px){
+        display: none;
+    }
+`
+
+export const ElemSetTitle = styled.span`
+    color: ${MAIN_PAGE_GREY_TEXT};
+    font-family: 'Montserrat-500';
+    font-size: 12px;
+`
+
+export const ElemSetValue = styled.span`
+    color: black;
+    font-family: 'Montserrat-700';
+    font-size: 12px;
+`
+
+export const ProgressBarSet = styled.div`
+    display: flex;
+    align-items: center;
+
+    @media (max-width: 400px){
+        display: ${props => props.isMobile ? "flex" : "none"};
+    }
+`
+export const ProgressBarInfo = styled.div`
+    padding-right: 20px;
+`
+export const InfoTasksDone = styled.span`
+    color: black;
+    font-family: 'Montserrat-700';
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 17px;
+`
+export const InfoTasksPlanned = styled.span`
+    color: black;
+    font-family: 'Montserrat-300';
+    font-size: 14px;
+    font-weight: 700;
```

Comment:
Interesting decision) You imported a font with a weight of 300 and then rewrite it